### DISCUSSION TITLE
My school doesn't provide students email

### DISCUSSION THREAD

**Discussion by EnesAgo** [community/community#55582] [GitHub Education]
I got rejected 4 times because of "Attendees of this school must verify and use their academic email address." reason. Our school Yahya Kemal College Struga does not provide any academic email. I asked our vice principal [Taner Sever](https://github.com/tanersever20) to open a ticket to confirm the status of student emails and his ticket automatically got closed.

I've Submitted a screenshot of the class schedule (dated) and a picture of the enrollment letter.

You can contact our vice principal [Taner Sever](https://github.com/tanersever20) at taner.sever@ykc.edu.mk

**Comment by jge162** [comment]
Do you not have school transcripts showing current term, school info and your name? that should help it is not just email.

**Comment by ryanlua** [comment]
Hi @EnesAgo,

**You do now need an email for Global Campus but other documents proving you are who you say you are.** Have you tried official or unofficial school transcripts as mentioned by @jge162, your Student ID, or other such similar documents?

If none of this helps, you can contact GitHub for support at [Get help with GitHub Education](https://support.github.com/contact/education) for them to manually approve or provide you with how you can get into Global Campus.

**Reply by EnesAgo** [reply]
Hi @RyanLua ,

I have tried everything that you have mentioned but again because my school does not provide academic email github is automaticaly rejecting me.

![Screenshot_20230516-074010_Gmail.jpg](https://github.com/community/community/assets/71901535/a8d645f4-8d09-4c63-b651-c61332d3fbc4)

**Reply by jge162** [reply]
Your school shows on GitHub somehow of having one. Are you sure they do not have school email? Maybe reach out to IT dept.

**Reply by EnesAgo** [reply]
I reached the vice principal, they said they don't provide emails for students, they provide emails only for teachers

**Reply by jge162** [reply]
That is an issue you need to resolve because it appears your school has people registered with GitHub and they use school email. Otherwise why does it keep asking your for email??? So I do not know how you can be approved otherwise without one. You need further assistance than chatting here. Good luck, just keep trying I guess.
